### PR TITLE
docs, Update dev-guide with fresh k8s-1.17 deploy

### PR DESCRIPTION
**Pull request by oshoval**
This commit updates the examples from running the old
k8s-1.17.0 to a new k8s-1.17 provider.

This PR doesn't update the logic part of the documentation.
It does update the running examples, and add info of missing commands
such as `export KUBEVIRTCI_TAG`.

Signed-off-by: Or Shoval <oshoval@redhat.com>

### CODE REVIEW

File: cluster-up/cluster/k8s-1.17/dev-guide.md [hunk 1]
```diff
@@ -25,59 +25,63 @@ It gives us isolation advantage and state freezing of the needed components, all
...
-NAME     STATUS   ROLES    AGE   VERSION
-node01   Ready    master   83m   v1.17.0
+NAME     STATUS   ROLES           AGE   VERSION
+node01   Ready    master,worker   58s   v1.17.16-rc.0
 ```
 
 # Inner look of a deployed cluster
 We can connect to the node of the cluster by:
 ```
+[ -z "$KUBEVIRTCI_TAG" ] && export KUBEVIRTCI_TAG=$(curl -L -Ss https://storage.googleapis.com/kubevirt-prow/release/kubevirt/kubevirtci/latest)
```
Comment (alonSadan) [inline cluster-up/cluster/k8s-1.17/dev-guide.md:51]: is this only on k8s-1.17? 
on my  k8s 1.20  I was able to connect to the node without exporting `KUBEVIRTCI_TAG`?

```
[root@zeus09 kubevirt]# env | grep KUBEVIRTCI_TAG
[root@zeus09 kubevirt]# cluster-up/ssh.sh node01
2021/04/07 15:39:54 Waiting for host: 192.168.66.101:22
2021/04/07 15:39:54 Connected to tcp://192.168.66.101:22
Last login: Tue Apr  6 16:28:00 2021 from 192.168.66.2
```

**Review by alonSadan — COMMENTED** [review]
Thanks Or. One question added.

File: cluster-up/cluster/k8s-1.17/dev-guide.md (hunk 1 above)
Comment (oshoval) [reply cluster-up/cluster/k8s-1.17/dev-guide.md:51]: You are on kubevirt, which exports it in its own scripts, please check kubevirtci (this PR is of kubevirtci repo)

```
kubevirtci (master) $ ./cluster-up/ssh.sh node01
FATAL: either KUBEVIRTCI_TAG or KUBEVIRTCI_GOCLI_CONTAINER must be set
kubevirtci (master) $ 
kubevirtci (master) $ 
kubevirtci (master) $ nodes
NAME     STATUS   ROLES                  AGE    VERSION
node01   Ready    control-plane,master   110s   v1.20.5
node02   Ready    worker                 47s    v1.20.5
kubevirtci (master) $ 
```

Comment (alonSadan) [reply cluster-up/cluster/k8s-1.17/dev-guide.md:51]: got it. Thanks.

**Review by alonSadan — APPROVED** [review]

**Comment by oshoval** [comment]
Thanks Alon

/cc @qinqon @fgimenez
Can you take a look please?

**Review by RamLavi — APPROVED** [review]
/lgtm

**Review by fgimenez — APPROVED** [review]
/lgtm
/approve

**Comment by oshoval** [comment]
Thanks

/retest

**Pull request merged**